explainer.md

```diff
@@ -197,9 +197,11 @@ If the system's underlying XR device changes (signaled by the `devicechange` eve
 
 The WebXR Device API provides information about the current frame to be rendered via the `XRFrame` object which developers must examine each iteration of the render loop. From this object the frame's `XRViewerPose` can be queried, which contains the information about all the views which must be rendered in order for the scene to display correctly on the XR device.
 
-`XRWebGLLayer` objects are not updated automatically. To present new frames, developers must use `XRSession`'s `requestAnimationFrame()` method. When the callback function is run, it is passed both a timestamp and an `XRFrame` containing fresh rendering data that must be used to draw into the `XRWebGLLayer`s `framebuffer` during the callback.
+`XRWebGLLayer` objects are not updated automatically. To present new frames, developers must use `XRSession`'s `requestAnimationFrame()` method. When the `requestAnimationFrame()` callback functions are run, they are passed both a timestamp and an `XRFrame`, which will contain fresh rendering data that must be used to draw into the `XRWebGLLayer`s `framebuffer` during the callback.
 
-`XRFrame` objects act as snapshots of the state of the XR device and all associated inputs. The state may represent historical data, current sensor readings, or a future projection. Due to it's time-sensitive nature, an `XRFrame` is only valid during the execution of the callback that it is passed into, and once control is returned to the browser any active `XRFrame` objects are marked as inactive. Calling any method of an inactive `XRFrame` will throw a [`NotAllowedError`](https://heycam.github.io/webidl/#notallowederror).
+A new `XRFrame` is created for each batch of `requestAnimationFrame()` callbacks. `XRFrame` objects act as snapshots of the state of the XR device and all associated inputs. The state may represent historical data, current sensor readings, or a future projection. Due to it's time-sensitive nature, an `XRFrame` is only valid during the execution of the callback that it is passed into, and once control is returned to the browser any active `XRFrame` objects are marked as inactive. Calling any method of an inactive `XRFrame` will throw a [`NotAllowedError`](https://heycam.github.io/webidl/#notallowederror).
```

Comment:
The first sentence here implies that `XRFrame`s are basically created only for rAF callbacks, while another key source is input event callbacks.  This is key context for the third sentence, which talks about "historical data", as the rAF `XRFrame` instances will all be about a future-predicted photon time.

Reply:
Good catch. Updated that line to say:

> A new `XRFrame` is created for each batch of `requestAnimationFrame()` callbacks or for certain events that are associated with tracking data.